pkg/sdk/row_access_policies_gen.go

```diff
@@ -29,7 +30,7 @@ type CreateRowAccessPolicyOptions struct {
 }
 
 type CreateRowAccessPolicyArgs struct {
-	Name string   `ddl:"keyword,no_quotes"`
+	Name string   `ddl:"keyword,double_quotes"`
```

Comment:
won't this be a breaking change for the users? I mean, it is added in the migration guide, that now it will be case-sensitive but what will happen with the policies created before? They are all uppercase now in Snowflake, right? Shouldn't we then advise to change all the names in configs to uppercase because of this change? Can we change what happens: if we do, and if we don't?

Reply:
Right, they are upper case now. Added a suggestion to the migration guide.

Reply:
Please also add that they will be migrated to the uppercase in state automatically

Reply:
Done.